README.md

```diff
@@ -11,15 +11,20 @@ This project is an implementation of a simple LC-3 (Little Computer 3) virtual m
 
 1. Clone the repository.
 
-2. Run the VM.
-    
-    You can run the VM with one or more LC-3 object files as arguments. Replace `object-file1` and `object-file2` with the paths to your LC-3 object files.
+2. Install the VM.
+    ```bash
+    cargo install --path .
+    ```
+
+3. Run the VM.
 
+    You can run the VM with one or more LC-3 object files as arguments. 
     ```bash
-    cargo run object-file1 object-file2 ...
+    lc3-vm object-file1 object-file2 ...
```

Comment:
I think we could add an explanation of how to pass from one program to another, just tried it and after one program ends it just returns to the terminal

Reply:
I think the problem is when we start the program, we call the `read_image_file` function with all the `paths` from the `main`, and it loads everything in the same memory space, using only the last object-file

Reply:
I think as the programmer you are supposed to account for that when writing the assembly. Specifically, with the assembler directives (indicating where the program begins in memory).

Reply:
Oh, didn't think about that case, then there is no problem 🤝

Reply:
> I think as the programmer you are supposed to account for that when writing the assembly. Specifically, with the assembler directives (indicating where the program begins in memory).

Yes, that's the idea.